### ISSUE TITLE
Symbols in /usr/lib/debug aren't automatically loaded

### ISSUE THREAD

**Issue by lucianposton** [radare/radare2#8666]
See https://github.com/radare/radare2/issues/4888 for context.

Several linux systems installs split debug symbols from executables and installs them under `/usr/lib/debug/` e.g. `/usr/path/to/bin/cmd` has symbols at `/usr/lib/debug/usr/path/to/bin/cmd`. radare2 1.6.0 doesn't appear to load these symbols. The symbols for calls are missing, attemping to seek, `s`, to a function fails, etc. On the other hand, gdb automatically loads the symbols from these files.

**Comment by lucianposton** [comment]
Also, `ob /usr/lib/debug/usr/path/to/bin/cmd` doesn't appear to manually load the symbols either.

**Comment by XVilka** [comment]
Code from https://github.com/radare/radare2/blob/master/libr/core/linux_heap_glibc.c#L213 should be generalized for using anywhere in RBin.

**Comment by radare** [comment]
same goes for jemalloc

**Comment by lucianposton** [comment]
In the meantime, is there a workaround to manually load symbols in version 2.0.0?

`o /usr/lib/debug/usr/path/to/bin/cmd` doesn't work. After issuing the preceding command, there is no error output, no new files are listed by `o`, and no symbols appear to have been loaded.

**Comment by tbarabosch** [comment]
@lucianposton I care too know as well. Did you find a solution?

**Comment by radare** [comment]
.!rabin2 -r ....

But you can use the ob subcommands too

**Comment by lucianposton** [comment]
> I care too know as well. Did you find a solution?

Nope. The ob command didn't load any symbols as far as I could tell.

I haven't yet tried the .!rabin2 -r ... suggestion above.

**Comment by radare** [comment]
Ob by itself doesnt do anything. Check the help message

**Comment by lucianposton** [comment]
oops. I meant to say `o /usr/lib/debug/usr/path/to/bin/cmd` didn't load the symbols, and `ob`, which is supposed to list opened files, did lis anything new after `o`.

**Comment by radare** [comment]
Nope, o opens a binary, replacing the current one, so nope

Ob doesnt lists the opened files. It lists the bin objects and u can use it to load the bin info of that bin on top of the current opened file

Also, historically this has been done via .!rabin2... since the very first versions of r1.

Which distro are u using to have test this?

**Comment by radare** [comment]
related to the dmi issue

**Comment by agausmann** [comment]
A workaround I've found is to use `eu-unstrip` from elfutils:

```
eu-unstrip /usr/bin/cmd /usr/lib/debug/usr/bin/cmd -o cmd.dbg
```
